src/main/java/at/hannibal2/skyhanni/features/garden/GardenYawAndPitch.kt

```diff
@@ -57,4 +58,14 @@ class GardenYawAndPitch {
     fun onGardenToolChange(event: GardenToolChangeEvent) {
         lastChange = SimpleTimeMark.farPast()
     }
+
+    @SubscribeEvent
+    fun onConfigFix(event: ConfigUpdaterMigrator.ConfigFixEvent) {
+        event.move(18, "garden.yawPitchDisplay.everywhere", "garden.yawPitchDisplay.showOutsideGarden")
+    }
+
+    private fun isEnabled() = (
+            (OutsideSbFeature.YAW_AND_PITCH.isSelected() && !LorenzUtils.inSkyBlock) ||
```

Comment:
why do you check if not in skyblock, seems like a redundant check

Reply:
bc outside sb doesn't make it show outside garden (like in hub)